lib/fs.js

```diff
@@ -89,6 +89,8 @@ function checkParentSync(path) {
 function writeFile(path, data, options, callback) {
   if (!path) throw new TypeError('path is required!');
 
+  if (!data) data = '';
```

Comment:
In the test cases we use `writeFile(path)` to create an empty file (leave `data` as an undefined value) will no longer be accepted by Node.js 14.